css/styles.css

```diff
@@ -4,11 +4,15 @@
 body {
   margin: 0;
   font-family: Roboto, sans-serif;
-  font-size: 0.7vw;
+  font-size: 0.7rem;
 }
   
 main {
   display: grid;
+  grid-template-areas: "header header header header"
+                       "globe description description selection"
+                       "globe donut1 donut2 small-globe"
+                       "globe lollipop lollipop lollipop";
```

Comment:
By using `grid-template-areas`, you can name your regions and simply re-layout those regions for different size screens (for example, see [here](https://github.com/emilyzhou112/dashboard-project/pull/1/files#diff-f6e4f9cac7473cf1fe513beba97669f195a6e1872bbd857c713d41e0f4b5d289R317-R325)). Then you don't have to worry about putting your interface components into specific columns and rows.